src/game/overlay.c

```diff
@@ -12,6 +12,8 @@
 
 #include <stdio.h>
 
+#include "log.h"
```

Comment:
I don't think we need this include, and if we do, it should be put with the rest of the `""` includes.